BetterMD/elements/table.py

```
@@ -1,113 +1,487 @@
+
     html = "th"
-    md = TdMD() 
-    rst = ThRST()
\ No newline at end of file
+    md = ThMD()
+    rst = ThRST()
+
+    children:'List[Text]' = List()
+    row:'Tr' = None  
+
+    def __init__(self, styles: dict[str, str] = {}, classes: list[str] = [], dom: bool = True, inner: list[Symbol] = [], **props):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)